### PR TITLE
[tests] fix NPE caused by oblogclient in OceanBase tests

### LINKED ISSUE #989: [test] OceanBaseConnectorITCase is stuck on self-hosted pipeline
https://dev.azure.com/ververica/flink-cdc-connectors/_build/results?buildId=1343&view=logs&j=c424a490-8b2d-5aa2-4797-106b8ec9494a

### PR DESCRIPTION
**Pull request by whhe**
Try with snapshot version, will replace with release version after ci passes, fix #989

### CODE REVIEW

**Comment by whhe** [comment]
@leonardBang The NPE issue should be fixed by this update, you can re-run the test job for OceanBase to verify it.

File: flink-sql-connector-oceanbase-cdc/pom.xml [hunk 1]
```diff
@@ -64,12 +64,9 @@ under the License.
                                     <include>commons-codec:commons-codec</include>
                                     <include>org.lz4:lz4-java</include>
                                     <include>org.apache.avro:avro</include>
-                                    <include>org.apache.commons:*</include>
+                                    <include>org.apache.commons:commons-lang3</include>
                                     <include>org.apache.kafka:*</include>
                                     <include>com.fasterxml.*:*</include>
-                                    <include>com.google.guava:*</include>
-                                    <!--  Include fixed version 18.0-13.0 of flink shaded guava  -->
-                                    <include>org.apache.flink:flink-shaded-guava</include>
```
Comment (leonardBang) [inline flink-sql-connector-oceanbase-cdc/pom.xml:72]: This is for compatible with Flink 1.14 ? why we change this

Comment (whhe) [reply flink-sql-connector-oceanbase-cdc/pom.xml:72]: `com.google.guava:guava` and `org.apache.commons:commons-collections4` are removed in oblogclient 1.0.3 , they are not required now. https://github.com/oceanbase/oblogclient/pull/24

Comment (leonardBang) [reply flink-sql-connector-oceanbase-cdc/pom.xml:72]: but why you remove the flink shaeded guava?

Comment (whhe) [reply flink-sql-connector-oceanbase-cdc/pom.xml:72]: Sure, will do it in next week.

**Pull request merged**